services/image_builder.go

```diff
@@ -352,56 +338,24 @@ echo "Done"
 		},
 	}
 
-	if !privileged {
-		envs = append(envs, corev1.EnvVar{
-			Name:  "BUILDKITD_FLAGS",
-			Value: "--oci-worker-no-process-sandbox",
-		})
-	}
-
+	var command []string
 	args := []string{
-		"build",
-		"--frontend",
-		"dockerfile.v0",
-		"--local",
-		"context=/workspace/buildcontext",
-		"--local",
-		fmt.Sprintf("dockerfile=%s", filepath.Dir(dockerFilePath)),
-		"--output",
-		fmt.Sprintf("type=image,name=%s,push=true,registry.insecure=%v", imageName, !opt.DockerRegistry.Secure),
-	}
-
-	annotations := make(map[string]string, 1)
-	if !privileged {
-		annotations["container.apparmor.security.beta.kubernetes.io/builder"] = "unconfined"
+		"--context=/workspace/buildcontext",
+		"--verbosity=info",
+		fmt.Sprintf("--dockerfile=%s", dockerFilePath),
+		fmt.Sprintf("--insecure=%v", !opt.DockerRegistry.Secure),
+		fmt.Sprintf("--destination=%s", imageName),
 	}
 
-	image := "quay.io/bentoml/buildkit:master-rootless"
-	if privileged {
-		image = "quay.io/bentoml/buildkit:master"
-	}
-
-	securityContext_ := &corev1.SecurityContext{
-		SeccompProfile: &corev1.SeccompProfile{
-			Type: corev1.SeccompProfileTypeUnconfined,
-		},
-		RunAsUser:  utils.Int64Ptr(1000),
-		RunAsGroup: utils.Int64Ptr(1000),
-	}
-	if privileged {
-		securityContext_ = &corev1.SecurityContext{
-			Privileged: utils.BoolPtr(true),
-		}
-	}
+	builder_image := "gcr.io/kaniko-project/executor:latest"
 
 	podsCli := kubeCli.CoreV1().Pods(kubeNamespace)
 
 	pod = &corev1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        kubeName,
-			Namespace:   kubeNamespace,
-			Labels:      kubeLabels,
-			Annotations: annotations,
```

Comment:
Need to retain annotations

Reply:
should I add this annotation empty annotation like `annotations := make(map[string]string, 1)`? or anything specific? is something else handling the annotations?